keel-api/src/main/kotlin/com/netflix/spinnaker/keel/api/artifacts/DeliveryArtifact.kt

```diff
@@ -93,5 +93,7 @@ abstract class DeliveryArtifact {
       )
     ).normalized()
 
+  fun toLifecycleRef() = "$deliveryConfigName:$reference"
```

Comment:
what does `toLifecycleRef` stands for?

Reply:
it's the key i'm using to store events, basically a reference to the artifact for lifecycle events